auth/postgres/domains.go

```diff
@@ -370,6 +370,17 @@ func (repo domainRepo) DeletePolicies(ctx context.Context, pcs ...auth.Policy) (
 
 	for _, pc := range pcs {
 		q := `
+		DELETE FROM
+			policies
+		WHERE
```

Comment:
Try and make this section DRY. You can use this as an example https://github.com/rodneyosodo/magistrala/blob/9f2fa81532db77ce790db3e5a938e7188065f73c/activitylog/postgres/events.go#L46C1-L50C130